addons/website_sale_wishlist/static/tests/tours/website_sale_wishlist_admin.js

```diff
@@ -0,0 +1,124 @@
+odoo.define('website_sale_wishlist_admin.tour', function (require) {
+'use strict';
+
+var rpc = require('web.rpc');
+var tour = require("web_tour.tour");
+
+tour.register('shop_wishlist_admin', {
+    test: false,
+    url: '/shop',
+},
+    [
+        {
+            content: "Create a product with always attribute and its values.",
+            trigger: 'body',
+            run: function () {
+                rpc.query({
+                    model: 'product.attribute',
+                    method: 'create',
+                    args: [{
+                        'name': "color",
+                        'display_type': 'color',
+                        'create_variant': 'always'
+                    }],
+                }).then(function (attributeId) {
+                    return rpc.query({
+                        model: 'product.template',
+                        method: 'create',
+                        args: [{
+                            'name': "Rock",
+                            'is_published': true,
+                            'attribute_line_ids': [[0, 0, {
+                                'attribute_id': attributeId,
+                                'value_ids': [
+                                    [0, 0, {
+                                        'name': "red",
+                                        'attribute_id': attributeId,
+                                    }],
+                                    [0, 0, {
+                                        'name': "blue",
+                                        'attribute_id': attributeId,
+                                    }],
+                                    [0, 0, {
+                                        'name': "black",
+                                        'attribute_id': attributeId,
+                                    }],
+                                ]
+                            }]],
+                        }],
+                    });
```

Comment:
Should do all this in python to speed up the test ideally
Meaning it won't work in browser dev tool without commiting the python lines once, but I guess it is already the case no, due to name unique constraint, the tour can only be ran once in the dev tool?